### DISCUSSION TITLE
"Fatal Error" after EME Upgrade (NOT URGENT)

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#76] [General]
On a test site, I upgraded EME version 2.3.14 directly to 2.3.52 by installing the zip from github. All seemed well but received a "critical error" trying to edit a country (CA). Everything else seems to be okay but I haven't checked everything. Before I upgrade my production site, I'd like to get this resolved but it is not urgent. Below is a snip of the email I received for my test site. Thanks in advance. / Roy

---------- SNIP -----------
In this case, WordPress caught an error with one of your plugins, Events Made Easy.

First, visit your website (https://facilitators.seniorshelpingseniors.ca/) and check for any visible issues. Next, visit the page where the error was caught (https://facilitators.seniorshelpingseniors.ca/wp-admin/admin.php?page=eme-countries&eme_admin_action=edit_country&id=1&eme_admin_nonce=4cfbc0dd98) and check for any visible issues.

Please contact your host for assistance with investigating this issue further.

If your site appears broken and you can't access your dashboard normally, WordPress now has a special "recovery mode". This lets you safely login to your dashboard and investigate further.

https://facilitators.seniorshelpingseniors.ca/wp-login.php?action=enter_recovery_mode&rm_token=vKrguXBZMduOHAhfQCV2FK&rm_key=4rbwwM6lA1vSkFvNt6OyNb

To keep your site safe, this link will expire in 1 day. Don't worry about that, though: a new link will be emailed to you if the error occurs again after it expires.

When seeking help with this issue, you may be asked for some of the following information:
WordPress version 6.1.1
Current theme: Septera (version 1.5.1)
Current plugin: Events Made Easy (version 2.3.52) PHP version 7.4.33

Error Details
=============
An error of type E_ERROR was caused in line 596 of the file /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-content/plugins/events-made-easy/eme-countries.php. Error message: Uncaught Error: Call to undefined function _esc_attr() in /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-content/plugins/events-made-easy/eme-countries.php:596
Stack trace:
#0 /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-content/plugins/events-made-easy/eme-countries.php(261): eme_countries_edit_layout()
#1 /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-includes/class-wp-hook.php(308): eme_countries_page('')
#2 /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#3 /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#4 /home/coscowor/facilitators.seniorshelpingseniors.ca/wp-admin/admin.php(259): do_action('events-made-eas...')
#5 {main}
  thrown

**Comment by liedekef** [comment]
Indeed, _esc_attr should be without the underscore. See this fix:
https://github.com/liedekef/events-made-easy/commit/62abb1001069130ce8d65565aedb23e61c600169

**Reply by roysinn** [reply]
Much better. Thanks for the quick fix, Franky.
